project/ReleaseUtils.scala

```diff
@@ -309,14 +309,23 @@ object ReleaseUtils {
        |build() {
        |  mkdir channel
        |  mv "$coursierChannelName" "channel/bloop.json"
-       |  coursier install --install-dir "$$srcdir" --default-channels=false --channel channel bloop
+       |  coursier install --install-dir "$$srcdir" --default-channels=false --channel channel --only-prebuilt=true bloop
        |}
        |
        |package() {
        |  cd "$$srcdir"
        |
-       |  ## binaries
-       |  install -Dm755 bloop "$$pkgdir"/usr/bin/bloop
+       |  # patch the bloop launcher so that it works when symlinked from /usr/bin
+       |  sed 's|$$(dirname "$$0")|/usr/lib/bloop|' -i bloop
+       |
+       |  # install to /usr/lib/bloop
+       |  # NOTE: bloop is just a launcher, the actual program is .bloop.aux
```

Comment:
@alexarchambault Are you aware of coursier install doing this in Linux machines? This is not how `coursier install` works in Windows and macOS 🤔

Reply:
For information, the "launcher" looks like this:
```
#!/usr/bin/env sh
export CS_NATIVE_LAUNCHER="true"
export IS_CS_INSTALLED_LAUNCHER="true"
exec "$(cd "$(dirname "$0")"; pwd)/.bloop.aux" "$@"
PK...
```
(then some binary data)

Reply:
> This is not how `coursier install` works in Windows and macOS

It should work this way there too. It's mostly there so that the script here can also act as a zip file, containing metadata about the installed app (how to update it, etc.), just like for JVM launchers.

Reply:
Thanks for the clarification @alexarchambault